sdk/nodejs/runtime/settings.ts

```diff
@@ -25,6 +26,9 @@ const resrpc = require("../proto/resource_grpc_pb.js");
 const resproto = require("../proto/resource_pb.js");
 const structproto = require("google-protobuf/google/protobuf/struct_pb.js");
 
+const maxRPCMessageSize: number = 1024 * 1024 * 400;
+const grpcChannelOptions = <ChannelOptions>{"grpc.max_receive_message_length": maxRPCMessageSize};
```

Comment:
Is the cast needed?  Or is this an option?

```suggestion
const grpcChannelOptions: ChannelOptions = {"grpc.max_receive_message_length": maxRPCMessageSize};
```

Reply:
ah yup totally, typescript n00b moment.